src/Symfony/Component/Console/Tests/Helper/OutputWrapperTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Tests\Helper;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Helper\OutputWrapper;
+
+class OutputWrapperTest extends TestCase
+{
+    public function testBasicWrap()
```

Comment:
since this is new code strict types can be added:

```suggestion
    public function testBasicWrap(): void
```